client/api_client.go

```diff
@@ -13,6 +13,7 @@ type ApiClient struct {
 
 type ApiClientInterface interface {
 	ConfigurationVariables(scope Scope, scopeId string) ([]ConfigurationVariable, error)
```

Comment:
Maybe worth changing existing method to `ConfigurationVariablesByScope` ?
No biggie, your call

Reply:
a66cb455(rename)